### DISCUSSION TITLE
JAVA CERTIFICATION COURSE

### DISCUSSION THREAD

**Discussion by rohitdeka-1** [community/community#147223] [Programming Help] [answered]
### Select Topic Area

Question

### Body

Is there any **JAVA Certification course** (_free_) with the **student developer pack** ? Please let me know  : )

**Comment by davevad93** [comment, marked as answer]
Hi @rohitdeka-1 ,

You can explore what the `Student Developer Pack` offers by going to the [pack offers page](https://education.github.com/pack). As far as I know, there's no `Java` certification course, but there's `Educative` which provides 6 months free access to over 70 hands-on courses, covering Web Development, Python, Java, and Machine Learning.

<img width="304" alt="5" src="https://github.com/user-attachments/assets/3df6cf71-1ed6-412a-aafe-a5958422f95b" />

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @rohitdeka-1!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!
